src/app/shared/card/card-prescriptions/card-prescriptions.component.html

```diff
@@ -0,0 +1,20 @@
+<div *ngIf="data" class="card-treatment">
+  <div class="title-container">
+    <div class="title">{{ drugInfo[0] }}</div>
+  </div>
+  <div class="date">{{ date | date: 'd/m/yyyy' }}</div>
```

Comment:
for the formats of the date we will need a constant so we can reuse it throut the project, 'd/m/yyyy'

Reply:
Added constants for the date formats I was using.